src/Modules/OrchardCore.Commerce/Drivers/PriceVariantsPartDisplayDriver.cs

```diff
@@ -50,7 +55,8 @@ public class PriceVariantsPartDisplayDriver : ContentPartDisplayDriver<PriceVari
                 viewModel => viewModel.VariantsCurrencies))
         {
             // Remove any content or the variants would be merged and not be cleared
```

Comment:
This comment is giving me a headache. Can you rewrite it?

Reply:
Your headache is understandable I couldn't really understand this either.